training-tracker-service/src/test/java/org/tolking/training_event_service/TrainingEventServiceImplTest.java

```diff
@@ -0,0 +1,112 @@
+package org.tolking.training_event_service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.modelmapper.ModelMapper;
+import org.tolking.training_event_service.dto.TrainingEventDTO;
+import org.tolking.training_event_service.entity.TrainingEvent;
+import org.tolking.training_event_service.enums.ActionType;
+import org.tolking.training_event_service.model.TrainerSummary;
+import org.tolking.training_event_service.repository.TrainerRepository;
+import org.tolking.training_event_service.service.impl.TrainingEventServiceImpl;
+
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.*;
+
+class TrainingEventServiceImplTest {
+
+    @Mock
+    private TrainerRepository trainerRepository;
+
+    @Mock
+    private ModelMapper modelMapper;
+
+    @InjectMocks
+    private TrainingEventServiceImpl trainingEventService;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
```

Comment:
alternatively you could annotate your test class with `@ExtendWith(MockitoExtention.class)`